### PR TITLE
Add profile modal

### PR DESCRIPTION
**Pull request by G-Ray**

### CODE REVIEW

**Review by leventdem — APPROVED** [review]
ok.

File: public/locales/en.json [hunk 1]
```diff
@@ -119,5 +119,12 @@
   "Synchronization failure": "Synchronization failure",
   "You can choose from": "You can choose from",
   "1 special character ": "1 special character ",
-  "Add a new device": "Add a new device"
+  "Add a new device": "Add a new device",
+  "This profile is already synchronized on this device.": "This profile is already synchronized on this device.",
+  "Add a new profile": "Add a new profile",
```
Comment (sujeebant) [inline public/locales/en.json:123]: Maybe we can replace "Add a new profile" by "Create a new profile" ?

Comment (G-Ray) [reply public/locales/en.json:123]: For now we will stick to the mockups

**Review by leventdem — APPROVED** [review]
OK, tested.

**Pull request merged**